### PR TITLE
My Portfolio Pull Request 

### PR DESCRIPTION
**Pull request by henask12**
in this pull request 
Set up HTML and CSS linter
Add global styles to ensure 1rem equals 10px
Create header and introductory section structure in the main HTML file
Add styles for tablets and desktops at the min break point of 768px
Thank you for reviewing my pull request

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @henask12 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that does not have a PR description and a well documented Readme File

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by henask12 — COMMENTED** [review]
i am sorry i forgot to put the description i will add it now, the read me is a document i believe is what is missing from the read me. and am i going to do new pull request or what

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          <a href="#" ><img src="./images/M.png" alt="M icon" /></a>
+        </li>
+        <li>
+          <a href="https://github.com/henask12" ><img src="./images/github.svg" alt="github icon" /></a>
+        </li>
+        <li>
+          <a href="#" ><img src="./images/hand.svg" alt="hand icon" /></a>
+        </li>
+      </ul>
+    </section>
```
Comment (Kamba56) [inline index.html:64]: - [ ] The background color of the header and headline section is white, kindly make the headline section white to match the figma template. You are expected to stick to the design template as much as possible.

- [ ] Kindly add the background image on the headline section to match the given template. You are expected to stick to the design template as much as possible, this includes colors and background images.
### Your design
image

### Figma
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              <a href="#" >Contact</a>
+            </li>
+          </ul>
+        </div>
+        <img class="navigation-bar_hamburger" src="./images/hamburguer.png" alt="hamburguer icon" />
+      </nav>
+    </header>
+
+    <section class="intro-section flex">
+      <h1 class="intro-section_title ">
```
Comment (Kamba56) [inline index.html:37]: - [ ] I noticed the font size of your h1 looks smaller than that of the figma template. Kindly fix this please.
## your design
image
### Figma
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @henask12, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using flexbox to layout contents of your page👏👏👏
- [x}

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (henask12) [reply index.html:37]: Resolved, Thank you

File: index.html (hunk 1 above)
Comment (henask12) [reply index.html:64]: Resolved

**Review by Owusu-Desmond — APPROVED** [review]
Hi @henask12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**